internal/store/store.go

```diff
@@ -11,6 +11,9 @@ import (
 	_ "github.com/lib/pq"
 )
 
+// ErrNoResult indicates that no data matching the query was found.
+var ErrNoResult = fmt.Errorf("no results found")
```

Comment:
Might be nice if ErrNoResult was actually a type instead of just a variable, so you could keep the context of the original error. https://play.golang.org/p/qc7fuTEsQX3